webapp/src/utils/tooltip_utils.ts

```diff
@@ -0,0 +1,23 @@
+export const validateGitlabURL = (url: string): boolean => {
+    const gitlabRegexPattern = /https?:\/\/(www\.)?.*\/([\w.?-]+)\/([\w-]+)\/-\/([\w-]+)\/([\d-]+$)/g;
+    return gitlabRegexPattern.test(url);
+};
+
+export const isValidUrl = (urlString: string): boolean => {
+    try {
+        return Boolean(new URL(urlString));
+    } catch (e) {
+        return false;
+    }
+};
+
+export const getTruncatedText = (text: string, length: number): string => {
+    if (text) {
+        let truncatedText = text.substring(0, length).trim();
```

Comment:
```suggestion
        let truncatedText = text.substring(length);
```

Reply:
will this work?

Reply:
@ayusht2810 That will make it truncate the beginning, instead of truncating the end